### PR TITLE
Move flags for rpc, messagebus and http enabled to container cluster

### PR DESCRIPTION
**Pull request by hmusum**

### CODE REVIEW

File: config-model/src/main/java/com/yahoo/vespa/model/container/Container.java [hunk 1]
```diff
@@ -286,16 +281,16 @@ public int getHealthPort()  {
...
         }
     }
 
     public String getStartupCommand() {
         return "PRELOAD=" + getPreLoad() + " exec vespa-start-container-daemon " + getJvmArgs() + " ";
     }
 
-    public boolean isRpcServerEnabled() {
-        return rpcServerEnabled;
+    private boolean isRpcServerEnabled() {
```
Comment (gjoranv) [inline config-model/src/main/java/com/yahoo/vespa/model/container/Container.java]: Does this class need both isRpcServerEnabled() and rpcServerEnabled()?

File: config-model/src/main/java/com/yahoo/vespa/model/container/xml/ContainerModelBuilder.java [hunk 2]
```diff
@@ -173,6 +174,8 @@ private void addClusterContent(ContainerCluster cluster, Element spec, ConfigMod
         addLegacyFilters(spec, cluster);  // TODO: Remove for Vespa 7
 
         addAthensCopperArgos(cluster, context);  // Must be added after nodes.
+        cluster.setRpcServerEnabled(rpcServerEnabled);
```
Comment (gjoranv) [inline config-model/src/main/java/com/yahoo/vespa/model/container/xml/ContainerModelBuilder.java]: These set statements don't really belong in addClusterContent. Move up to doBuild() or create a new method.

**Review by gjoranv — CHANGES_REQUESTED** [review]
Great stuff! Just a couple of comments.

**Comment by hmusum** [comment]
@gjoranv Code updated as suggested, please take a look again

**Review by gjoranv — APPROVED** [review]

**Pull request merged**